graf2d/graf/src/TLegend.cxx

```diff
@@ -825,15 +825,26 @@ void TLegend::PaintPrimitives()
             dynamic_cast<TAttFill*>(eobj)->Copy(*entry);
          }
 
+         // Case of exclusion graphs
+         Float_t wl = 1., wu = 1.;
+         if (eobj && eobj->InheritsFrom(TAttLine::Class())
+                  && eobj->InheritsFrom(TGraph::Class())) {
+            Int_t w = dynamic_cast<TAttLine*>(eobj)->GetLineWidth();
+            if (TMath::Abs(w)>999) {
```

Comment:
@couet a minor point: https://root.cern.ch/doc/master/classTGraphPainter.html#GrP2 says
> This drawing mode is activated when the absolute value of the graph line width (set by [SetLineWidth()](https://root.cern.ch/doc/master/TGWin32VirtualXProxy_8cxx.html#a18a240c3a19b7b71d058ce1da94747e9)) is greater than 99

should this condition be changed to `>99` instead of `>999`?

Reply:
Perfectly right !

Reply:
PR here: https://github.com/root-project/root/pull/14389